vscode/src/chatParticipant.ts

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as vscode from "vscode";
+import { getRandomGuid } from "./utils";
+import { log } from "qsharp-lang";
+import { getAuthSession, scopes } from "./azure/auth";
+
+const chatUrl = "https://canary.api.quantum.microsoft.com/api/chat/completions";
```

Comment:
Did you chat with Zander about this? I assume we shoudln't be using a `canary*` endpoint when we ship.

Reply:
The endpoint has been updated appropriately. Thanks!